test/unit/vdom/properties.js

```diff
@@ -117,4 +117,36 @@ describe('vdom/properties', () => {
       );
     });
   });
+
+  it('#876 - Apply properties and attributes in a polyfill environment', (done) => {
+    const Elem1 = define('x-test', {
+      render (elem) {
+        return h(Elem2, { foo: true, bar: true, baz: true });
```

Comment:
Up until now I was sure that camelCased props transform into attributes with dashes. I.e. 'testProp' becomes 'test-prop'.
But in this test 'testProp' is transforming into 'testprop'. Why?